core-ui/src/components/BusolaExtensions/BusolaExtensionDetails.js

```diff
@@ -0,0 +1,188 @@
+
+    const currentVersion = formatCurrentVersion(configmap?.data?.version);
+    const hasMigrationFunction = getMigrationFunctions().some(
+      version => version === currentVersion?.replace('.', ''),
+    );
+    const isCurrentVersion = getLatestVersion() === currentVersion;
+    const isSupportedVersion = getSupportedVersions().some(
+      version => version === currentVersion,
+    );
+
+    const showMessage = () => {
+      if (isCurrentVersion) {
+        return null;
+      } else if (isSupportedVersion) {
+        return (
+          <MessageStrip type="information" className="fd-margin-bottom--sm">
+            {t('extensibility.message.old-version')}
+          </MessageStrip>
+        );
+      } else if (hasMigrationFunction) {
+        return (
+          <MessageStrip type="error" className="fd-margin-bottom--sm">
+            {t('extensibility.message.unsupported-version')}
+          </MessageStrip>
+        );
+      } else {
+        return (
+          <MessageStrip type="error" className="fd-margin-bottom--sm">
+            <Trans i18nKey="extensibility.message.unnown-version">
+              <Link
+                className="fd-link"
+                url="https://github.com/kyma-project/busola/tree/main/docs/extensibility"
+              />
+            </Trans>
+          </MessageStrip>
+        );
+      }
+    };
+    return (
+      <LayoutPanel className="fd-margin--md">
+        <LayoutPanel.Header>
+          <LayoutPanel.Head title={t('extensibility.section.version')} />
+          <LayoutPanel.Actions>
+            {hasMigrationFunction && (
+              <>
+                {currentVersion && (
+                  <Button
+                    disabled={currentVersion === getLatestVersion()}
+                    glyph="forward" // generate-shortcut journey-arrive journey-change tools-opportunity trend-up
+                    onClick={() => {
+                      const newBusolaExtension = migrateToLatest(configmap);
+                      updateBusolaExtension(newBusolaExtension, configmap);
+                    }}
+                  >
+                    {t('config-maps.buttons.migrate')}
+                  </Button>
+                )}
+              </>
+            )}
+          </LayoutPanel.Actions>
+        </LayoutPanel.Header>
+        <LayoutPanel.Body>
+          {showMessage()}
+          <LayoutPanelRow
+            name={t('extensibility.version.current')}
+            value={currentVersion || EMPTY_TEXT_PLACEHOLDER}
+          />
+          <LayoutPanelRow
+            name={t('extensibility.version.latest')}
+            value={getLatestVersion()}
+          />
+        </LayoutPanel.Body>
+      </LayoutPanel>
+    );
+  };
+
+  const customColumns = [
+    {
+      header: t('common.headers.owner'),
+      value: secret => (
+        <ControlledBy ownerReferences={secret.metadata.ownerReferences} />
+      ),
+    },
+  ];
+
+  return (
+    <ResourceDetails
```

Comment:
Navigation to list using the breadcrumb doesn't work due to `Navigation not possible, navigationContext configmaps not found.`

Reply:
fixed